doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,278 @@
+
+Astarte may declare to have no (or inconsistent) knowledge about device owned properties, this might happen after an error (Astarte should keep knowledge about a remote device status).
+Astarte performs this task by disconnecting and clearing the session.
+
+After a clean session properties might be purged.
+
+## Purge Properties
+
+Either a Device or Astarte may tell the remote host the set properties list. Any property that is not part of the list will be deleted from any cache or database.
+This task is called _purge properties_ in Astarte jargon, and it is performed by publishing a the list of known set properties to `/control/consumer/properties` or `/control/producer/properties`.
+
+Purge Properties payload is a zlib deflated plain text, with an additional 4 bytes header.
+The additional 4 bytes header is the size of the uncompressed payload, encoded as big endian uint32.
+
+Following example is a payload compressed using zlib default compression, with the additional 4 bytes header:
+```
+0000000 00 00 00 46 78 9c 4b ce cf d5 4b ad 48 cc 2d c8
+0000020 49 d5 f3 ad f4 cc 2b 49 2d 4a 4b 4c 4e d5 2f ce
+0000040 cf 4d d5 2f 48 2c c9 b0 ce 2f 4a 87 ab 70 29 4a
+0000060 4c 2b 41 28 ca 2f c9 48 2d 0a 00 2a 02 00 b2 0c
+0000100 1a c9
+```
+
+The uncompressed plain text payload has the following format (in BNF like syntax):
+
+```
+properties ::= properties_list
+properties_list ::= properties_entry ";" properties_list | properties_entry
+properties_entry ::= interface_name path
+```
+
+The following example is the inflated previous payload:
+
+```
+com.example.MyInterface/some/path;org.example.DraftInterface/otherPath
+```
+
+This protocol feature is fundamental when a device has any interface with an *allow_unset* mapping, *purge properties* allows to correct any error due to unhandled unset messages.
+
+## Publishing Data
+
+Either Astarte or a device might publish new data on a interface/endpoint specific topic.
+The topic is built using `/<interface name>/<path>` schema, and it is used regardless the type of interface or mapping.
+
+Also `/` path is a valid path for object aggregated interfaces.
+
+Following device topics are valid:
+
+* `/com.example.MyInterface/some/path`
+* `/org.example.DraftInterface/otherPath`
+* `/com.example.astarte.ObjectAggregatedInterface/`
+
+Data messages QoS is chosen according to mapping settings, such as reliability. Properties are always published using QoS 2.
+
+| Interface Type | Reliability     | QoS |
+|----------------|-----------------|-----|
+| properties     | always unique   | 2   |
+| datastream     | unreliable      | 0   |
+| datastream     | guaranteed      | 1   |
+| datastream     | unique          | 2   |
+
+### Payload Format
+
+Payload format might change according to the message type. Payloads are always BSON encoded, except for unset messages that are empty.
+
+#### Property Message
+
+Property messages have a "v" key (which means value) and they look like the followings:
```

Comment:
Rephrase, you specified this before:
Property messages support only ["value" among Astarte payload standard fields](#astarte-payload-standard-fields). Valid examples are: